src/ecc_key.c

```diff
@@ -1425,3 +1449,28 @@ static int ec_key_spki_has_valid_mkvp(const struct zpc_ec_key *ec_key,
 
 	return 0;
 }
+
+static int ec_key_blob_has_valid_mkvp(struct zpc_ec_key *ec_key, const unsigned char *buf)
+{
+	const unsigned char *mkvp;
+	unsigned int mkvp_len;
+
+	if (ec_key->mkvp_set == 0)
+		return 1; /* cannot judge */
+
+	if (ec_key->type == ZPC_EC_KEY_TYPE_CCA) {
+		mkvp = ((struct ccakeytoken *)buf)->mkvp;
+		mkvp_len = 8;
```

Comment:
Maybe use defines in the corresponding header ?
Same below for EP11

Reply:
OK, introduced 2 constants MKVP_LEN_CCA and MKVP_LEN_EP11.
Done.